### ISSUE TITLE
Feat: "Open Source Best Practices" for Students

### ISSUE THREAD

**Issue by yuluo-yx** [LinuxSuRen/open-source-best-practice#156]
Most of the open source best practices are aimed at people who already have some experience. For students with no development experience, how should they participate in the open source community?

I think I should write an article from a student perspective to guide them on how to participate in open source and grow.

**Comment by LinuxSuRen** [comment]
Totally agree with you. You can write it in this repository or your own. But it's worth to do no matter what. Actually, people might need different guide who have different background. For example:

* CS student
* non-CS student
* graduated 1 ~ 3 years
* and more

Hopefully we could have much helpful practice or guide for different people.

**Comment by yuluo-yx** [comment]
> Totally agree with you. You can write it in this repository or your own. But it's worth to do no matter what. Actually, people might need different guide who have different background. For example:
> 
> * CS student
> * non-CS student
> * graduated 1 ~ 3 years
> * and more
> 
> Hopefully we could have much helpful practice or guide for different people.

yep, agree with you. There is no big difference between computer science students and non-computer science students. They should be the same whole (technical and cognitive level). In my opinion, we can classify them as a whole.

**Comment by LinuxSuRen** [comment]
> There is no big difference between computer science students and non-computer science students.

I already forget the status when I was a student. For non-CS students, they should learn CS first, right? If we have a non-CS student contributors, they could contribute some references or expericens about it.

**Comment by yuluo-yx** [comment]
> > There is no big difference between computer science students and non-computer science students.
> 
> I already forget the status when I was a student. For non-CS students, they should learn CS first, right? If we have a non-CS student contributors, they could contribute some references or expericens about it.

This is a question worth discussing. 😄😄 
Maybe it needs to be discussed in groups.

**Comment by LinuxSuRen** [comment]
> Maybe it needs to be discussed in groups.

I recommend that discuss it base on a draft content which might be a PR. Please don't afraid of the draft content not being good enough. We can continously improve it.

**Comment by yuluo-yx** [comment]
> > Maybe it needs to be discussed in groups.
> 
> I recommend that discuss it base on a draft content which might be a PR. Please don't afraid of the draft content not being good enough. We can continously improve it.

It's a pleasure to participate in and contribute to the discussion.

I have time to write about some practices of computer science students participating in open source, and I will submit it later.
